.buildkite/pipeline.yml

```diff
@@ -8,8 +8,6 @@ steps:
       git status
       changed_files=$$(git status --porcelain --untracked-files=no | wc -l)
       if [ $$changed_files -gt 0 ]; then
-        yarn --version
-        git diff
```

Comment:
Why did buildkite stuck executing this lines?

Reply:
I suspect that Git tries to display the diff using `less`, which waits for user input.

Reply:
Yes @abacabadabacaba is right